app/workers/keybase_proof_worker.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class KeybaseProofWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: 'pull', retry: 5, unique: :until_executed
+
+  def perform(proof_id)
+    proof = AccountIdentityProof.keybase.with_account_username.find(proof_id)
+
+    validity_and_liveness = KeybaseProofService.new.call(
+      provider_username: proof.provider_username,
+      username: proof.username,
+      token: proof.token
```

Comment:
pass in `domain` here (even if it's `Keybase.local_domain` for now so we don't forget